build.sbt

```diff
@@ -41,7 +41,7 @@ addCommandAlias(
 )
 addCommandAlias(
   "testJVMDotty",
-  ";coreJVM/test:compile;stacktracerJVM/test:compile;streamsJVM/test:compile;testTestsJVM/run;testTestsJVM/test;testRunnerJVM/test:run;examplesJVM/test:compile"
+  ";coreJVM/test;stacktracerJVM/test;streamsJVM/test;testTestsJVM/run;testTestsJVM/test;testRunnerJVM/test:run;examplesJVM/test:compile"
```

Comment:
The tests are in a separate project, so to run the tests the command needs to be `coreTestsJVM/test`.